apps/server/src/routes/swagger/schemas.yml

```diff
@@ -2,45 +2,70 @@
 
 components:
   responses:
+    BadRequest:
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/responses/Error'
+
     UnauthorizedError:
       description: Access token is missing or invalid
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/CreateSubmissionResult'
+            $ref: '#/components/responses/Error'
 
     ServerError:
       description: Internal Server Error
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/CreateSubmissionResult'
+            $ref: '#/components/responses/Error'
 
     ServiceUnavailableError:
       description: Service Unavailable Error
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/CreateSubmissionResult'
+            $ref: '#/components/responses/Error'
+
+    Error:
+      type: object
+      properties:
+        error:
+          type: string
+          description: Error type
+        message:
+          type: string
+          decription: Description of the error
 
   schemas:
     CreateSubmissionResult:
       required:
-        - successful
-        - submission
+        - state
+        - description
         - batchErrors
+        - inProcessEntities
```

Comment:
not asking for changes, but out of curiosity:
does the order of these items change anything (was going to say "why not alphabetical?", but then realised there may be a reason)